libs/langchain/langchain/document_loaders/pdf.py

```diff
@@ -608,7 +608,11 @@ class DocumentIntelligenceLoader(BasePDFLoader):
     """Loads a PDF with Azure Document Intelligence"""
 
     def __init__(
-        self, file_path: str, client: Any, model: str = "prebuilt-document"
+        self,
+        file_path: str,
+        client: Any,
+        model: str = "prebuilt-document",
+        split_mode: str = "page",
```

Comment:
@baskaryan here's where it's defaulted so it won't introduce any breaking change.